Formula/z/zuban.rb

```diff
@@ -21,9 +21,9 @@ class Zuban < Formula
   def install
     system "cargo", "install", *std_cargo_args(path: "crates/zuban")
 
-    (lib/"typeshed").mkpath
-    cp_r Formula["mypy"].opt_libexec.glob("lib/python*/site-packages/mypy/typeshed")[0].children, lib/"typeshed"
-    bin.env_script_all_files(libexec/"bin", ZUBAN_TYPESHED: lib/"typeshed")
+    (libexec/"typeshed").mkpath
```

Comment:
Thank you, that was definitely unintentional. Let's put it `libexec/lib`

```suggestion
    (libexec/"lib/typeshed").mkpath
```